examples/api/test/widgets/shortcuts/callback_shortcuts.0_test.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter_api_samples/widgets/shortcuts/callback_shortcuts.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('CallbackShortcutsApp', () {
```

Comment:
`group` isn't necessary here.  As this test file only contains Callback Shortcuts tests.